qgreenland/config/layers/Glaciology/Ice sheet velocity/GrIMP/grimp.py

```diff
@@ -0,0 +1,179 @@
+                    "--type",
+                    "Int32",
+                    "-A",
+                    "{input_dir}/" + f"*{variable}*.tif",
+                    "--outfile",
+                    "{output_dir}/scaled.tif",
+                ],
+            ),
+            # The `scale` metadata lets tools like QGIS know that the integers
+            # should be interpreted as floating point data (e.g,. a value of
+            # `12345` is displayed as 123.45)
+            *gdal_edit(
+                input_file="{input_dir}/scaled.tif",
+                output_file="{output_dir}/edited.tif",
+                gdal_edit_args=[
+                    "-scale",
+                    "0.01",
+                ],
+            ),
+            *compress_and_add_overviews(
+                input_file="{input_dir}/edited.tif",
+                output_file="{output_dir}/compressed_with_overviews.tif",
+                dtype_is_float=False,
+            ),
+        ],
+    )
+
+
+_description_common = """
+Annual mosaics are produced from data with resolutions varying from a few
+hundred meters to 1.5km. The 2021 annual mosaic includes data collected between
+2020-12-01 and 2021-11-30.
+
+Note that these data have been rounded to the nearest centimeter for QGreenland
+to save disk space. Please see the original data source for the un-modified and
+additional data:
+
+* x and y component velocity error estimates (ex, ey).
+* A temporal offset parameter (dT) that reports the difference in days between
+  the date of each velocity estimate and the midpoint date of the corresponding
+  measurement period.
+* Shapefile that indicates the source of the image pairs (SAR or Landsat 8) used
+  to produce the mosaic.
+"""
+
+
+_layer_params = {
+    "vv": {
+        "description": "Ice sheet velocity magnitude (vv) in meters per year for 2021."
+        + _description_common,
+        "style": "grimp_velocity_magnitude",
+        "title": "Annual ice sheet velocity magnitude 2021 (200m)",
+    },
+    "vx": {
+        "description": "Ice sheet velocity x component (vy) in meters per year for 2021."
+        + _description_common,
+        "style": "grimp_velocity_component",
+        "title": "Annual ice sheet velocity x component 2021 (200m)",
+    },
+    "vy": {
+        "description": "Ice sheet velocity y component (vy) in meters per year for 2021."
+        + _description_common,
+        "style": "grimp_velocity_component",
+        "title": "Annual ice sheet velocity y component 2021 (200m)",
+    },
+}
+
+annual_grimp_layers = [
+    make_layer(
+        variable=variable,
+        layer_id=f"grimp_annual_{variable}_2021",
+        style=layer_params["style"],
+        description=layer_params["description"],
+        title=layer_params["title"],
+    )
+    for variable, layer_params in _layer_params.items()
+]
+
+grimp_vector_layer = Layer(
+    id="grimp_annual_vectors_2021",
+    title="Annual ice sheet velocity vectors 2021",
```

Comment:
Should we indicate the resolution of the source data in the title (1.5 km)? Usually we don't include the resolution for vector data, but this is derived from the downsampled vx and vy grids.

We do this for the RACMO wind vectors ("Annual mean wind vectors 1958-2019 (5km)")

Reply:
I think we should because this is gridded vector data. I think we have other gridded vector layers aside from RACMO, and I don't know if we do the same there, but I think we should :)